integration/index.ts

```diff
@@ -12,12 +12,14 @@ interface NetlifyCMSOptions {
    * Path at which the Netlify CMS admin dashboard should be served.
    * @default '/admin'
    */
+  disableIdentityWidgetInjection?: boolean;
   adminPath?: string;
   config: Omit<CmsConfig, 'load_config_file' | 'local_backend'>;
```

Comment:
The comment above this needs to be directly above the `adminPath` option as that’s what it’s describing. Let’s move this option down to keep these in alphabetical order:

```suggestion
  adminPath?: string;
  config: Omit<CmsConfig, 'load_config_file' | 'local_backend'>;
  disableIdentityWidgetInjection?: boolean;
```